doc/explanation/networks.md

```diff
@@ -1,7 +1,107 @@
+
+        lxc config device add <instance_name> <device_name> nic network=<network_name>
+
+  If needed, you can add further properties to the command to override the default settings for the network.
+
+## Managed networks
+
+Managed networks in LXD are created and configured with the `lxc network` command.
+
+Depending on the network type, LXD either fully controls the network or just manages an external network interface.
+
+Note that not all {ref}`NIC types <instance_device_type_nic>` are supported as network types.
+LXD can only set up some of the types as managed networks.
+
+### Fully controlled networks
+
+Fully controlled networks provide most functionality, including, for example, the ability to do IP management.
+
 LXD supports the following network types:
 
- - [bridge](#network-bridge): Creates an L2 bridge for connecting instances to (can provide local DHCP and DNS). This is the default.
- - [macvlan](#network-macvlan): Provides preset configuration to use when connecting instances to a parent macvlan interface.
- - [sriov](#network-sriov): Provides preset configuration to use when connecting instances to a parent SR-IOV interface.
- - [ovn](#network-ovn): Creates a logical network using the OVN software defined networking system.
- - [physical](#network-physical): Provides preset configuration to use when connecting OVN networks to a parent interface.
+{ref}`network-bridge`
+: A network bridge creates a connection between different networks or network segments.
+
+  In LXD context, the `bridge` network type creates an L2 bridge that connects the instances that use it and makes it possible to pass traffic between them.
+  The bridge can also provide local DHCP and DNS.
+
+  This is the default network type.
+
+{ref}`network-ovn`
+: {abbr}`OVN (Open Virtual Network)` is a software-defined networking system that supports virtual network abstraction.
+  You can use it to build your own private cloud.
+  See https://www.ovn.org/ for more information.
+
+  In LXD context, the `ovn` network type creates a logical network.
+  To set it up, you must install and configure the OVN tools.
+  In addition, you must create an uplink network that provides the network connection for OVN.
+  As the uplink network, you should use one of the external network types.
+
+### External networks
+
+External networks use network interfaces that already exist.
+Therefore, LXD has limited possibility to control them.
+
+The main purpose for using external networks is to provide an uplink network through a parent interface.
+
+LXD supports the following network types:
+
+{ref}`network-macvlan`
+: Macvlan is a virtual {abbr}`LAN (Local Area Network)` that can assign several IP addresses to the same network interface, basically splitting up the network interface into several sub-interfaces with their own IP addresses.
+  The IP addresses are assigned based on randomly generated MAC addresses.
```

Comment:
Random MAC addresses are not a property of any of our network types. They are a property of he NIC devices themselves, and the random generation of the NIC MAC address isn't specific to macvlan type NICs.

Reply:
Yes it is, because even general macvlan (outside from LXD) doesn't get involved with IP assignment at all.